src/basilisp/core.lpy

```diff
@@ -1158,11 +1158,9 @@
   (basilisp.lang.runtime/compare x y))
 
 (defn sort
-  "Return a sorted sequence of the elements from ``coll``\\.
+  "Return a sorted sequence of the elements from ``coll`` using the ```cmp``` comparator\\.
```

Comment:
```suggestion
  "Return a sorted sequence of the elements from ``coll`` using the ``cmp`` comparator\\.
```

These docstring changes make sense to me, but I think they should just be double backticks not triple.

Reply:
Done